modules/bs.environment/data/bs.environment/tags/function/get_moon_angle.json

```diff
@@ -0,0 +1,20 @@
+{
+  "__bookshelf__": {
+    "feature": true,
+    "documentation": "https://docs.mcbookshelf.dev/en/latest/modules/environment.html#get-moon-angle",
```

Comment:
This anchor is i think wrong and will direct to nothing should be celestial-movement (a valid title in the doc), same for all others